GLOSSARY.md

```diff
@@ -64,6 +64,10 @@ Block devices, HDDs, SDDs, or other types of storage devices addressable by `/de
 
 Commonly known as "the package manager for Kubernetes", [Helm](https://helm.sh/) is a CLI tool that builds and deploys Kubernetes applications through application packages known as Helm Charts.
 
+### IEM
+
+IEM stands for Interesting Event Messages, relating to events happening within the system, affecting normal system functionality and which may be useful for preliminary debugging. 
```

Comment:
I would want to add in some additional commentary acknowledging that "IEM" is a pretty common term throughout Cloud Computing et al (similar to the definition of "Node" below).

```suggestion
In the context of [CORTX](https://github.com/Seagate/cortx), IEM stands for Interesting Event Messages, relating to events happening within the system, affecting normal system functionality and which may be useful for preliminary debugging. 
```